### PR TITLE
fea: Skeleton restyle, Font Awesome Icons

### PR DESCRIPTION
**Pull request by matekdev**
- Installs font awesome icons
- Restyles skeleton to use crimson styling

image
image

### CODE REVIEW

File: apps/dashboard/src/routes/+layout.svelte [hunk 1]
```diff
@@ -29,16 +33,32 @@
...
+				>
 				<a href="/login"
-					><Avatar
-						src="https://images.unsplash.com/photo-1617296538902-887900d9b592?ixid=M3w0Njc5ODF8MHwxfGFsbHx8fHx8fHx8fDE2ODc5NzExMDB8&ixlib=rb-4.0.3&w=128&h=128&auto=format&fit=crop"
-						width="w-8"
-						rounded="rounded-full"
-					/>
-				</a>
+					><button type="button" class="btn variant-filled-surface">
+						{#if data.session}
```
Comment (matekdev) [inline apps/dashboard/src/routes/+layout.svelte:53]: Should I setup a store for this? I imagine there might be a few places where we want to check if a user is logged in. (Assuming that store is the correct use for what I want here)

Comment (MisterJimson) [reply apps/dashboard/src/routes/+layout.svelte:53]: Have you seen a Supabase project use a Store for this?

Comment (matekdev) [reply apps/dashboard/src/routes/+layout.svelte:53]: nope, was just an idea that I had.

Comment (MisterJimson) [reply apps/dashboard/src/routes/+layout.svelte:53]: Kk lets keep and eye on it and see how it would work as we add more pages

**Review by MisterJimson — APPROVED** [review]

**Pull request merged**